docs/service-loadbalancer.md

```diff
@@ -183,15 +183,33 @@ The Exoscale NLB service health checking retries before considering a target
 ### Using a Kubernetes Ingress Controller behind an Exoscale NLB
 
 If you wish to expose a Kubernetes [Ingress Controller][k8s-ingress-controller]
-(such as the popular [ingress-nginx][ingress-nginx]) behind an Exoscale NLB,
-some adjustments to its manifests are required in order for the CCM to be able
-to provision a corresponding NLB instance correctly:
-
-* Since the Ingress traffic will be load-balanced across multiple Instance Pool
-  members, the Ingress Controller must either run on every cluster Node (e.g.
-  using a [`DaemonSet`][k8s-daemonset] type of deployment), or the Ingress
-  Controller Service `externalTrafficPolicy` property must be set to `Cluster`
-  (not `Local`).
+(such as the popular [ingress-nginx][ingress-nginx]), or any other *Service*
+behind an Exoscale NLB, please note that by default the traffic is forwarded
+to all [healthy] Nodes in the destination Instance Pool, whether they actually
+host *Pods* targeted by the *Service* or not – which may result in additional
+hops inside the Kubernetes cluster, as well as losing the source IP address
+(source NAT). To prevent this, 2 configurations are possible:
+
+* Ensure that least one *Service* *Pod* runs on every *Node* of the cluster,
+  for example using a [*DaemonSet*][k8s-daemonset] instead of a *Deployment*
+  with a *Service* `spec.externalTrafficPolicy` set to `Cluster`. This ensures
```

Comment:
If this is set to Cluster, source NAT will happen, even if there is a pod on every node.